client/tests/phpunit/Service/DocumentServiceTest.php

```diff
@@ -320,17 +325,25 @@ public function testRetrieveDocumentsFromS3ByReportSubmissionsMissingDocs()
         self::assertEquals([$expectedMissingDoc1, $expectedMissingDoc2], $missing);
     }
 
-    public function testCreateMissingDocumentsFlashMessage()
+    public function testCreateMissingDocumentsFlashMessage(): void
     {
         $missingDoc = new MissingDocument();
         $missingDocuments = [$missingDoc];
 
         $expectedFlash = 'some flash message here';
 
+        /** @var ObjectProphecy|S3Storage $storage */
         $storage = self::prophesize(S3Storage::class);
+
+        /** @var ObjectProphecy|LoggerInterface $logger */
         $logger = self::prophesize(LoggerInterface::class);
+
+        /** @var ObjectProphecy|RestClient $restClient */
         $restClient = self::prophesize(RestClient::class);
+
+        /** @var ObjectProphecy|Environment $twig */
         $twig = self::prophesize(Environment::class);
```

Comment:
This test setup is repeated a few times above. Could we do this once in the `setUp()` block instead?

Reply:
Yep, good point. I've converted the class properties to use prophecy and rewritten the tests to use them rather than setting up their own mocks/subject